### ISSUE TITLE
Turing exports "plot" ? 

### ISSUE THREAD

**Issue by DarkSnail** [yebai/Turing.jl#155]
Not critical, but a bit annoying in the long run: it seems that Turing exports a "plot" function that conflicts with other plot packages. In my case...
```julia
using Turing
using Plots
gs()
```
I get 
```
WARNING: both Plots and Turing export "plot"; uses of it in module Main must be qualified
``` 
and I can plot only using ``` Plots.plot```

**Comment by xukai92** [comment]
Thank you for pointing this. I believe it's unnecessary for Turing to export `plot`. We will remove it soon.

**Comment by DarkSnail** [comment]
Thank you very much. Does Turin have a Gadfly dependency, too? I think it is superfluous.  Can I ask another thing? I have found something that maybe is a bug, maybe it is just due to my inexperience with probabilistic programming... I am not confident enough to open an issue here. Is there a mailing list or something along these lines?

**Comment by xukai92** [comment]
I don't think Turing has Gadfly dependency but we might use it somewhere in our benchmark script. Please feel free to report bugs in issues as Turing is still in its early stage so bugs are quite possible. Or alternatively you can ask things in Gitter (https://gitter.im/Turing-jl/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).

**Comment by yebai** [comment]
Yes Turing does depend on `Gadly`. This dependency on `Gadfly` is inherited from Mamba, which is used by `Turing` to summaries MCMC results. 

Actually, the `plot` export is also from `Mamba`, which allows user to plot trace plots for chains (e.g. `plot(chain)`).

**Comment by yebai** [comment]
> Is there a mailing list or something along these lines?

Yes, we maintain a mailing list for developers and users: turing-project@eng.cam.ac.uk

**Comment by DarkSnail** [comment]
Thanks! I will message you over there with a bare-bone example of the issue I have.

**Comment by xukai92** [comment]
@yebai For the `plot` thing, is it necessary for us to export `plot` in Turing (https://github.com/yebai/Turing.jl/blob/master/src/Turing.jl#L28) as well?

**Comment by yebai** [comment]
It won't cause issue, but we would need to write `Turing.plot(chain)`.
maybe we can remove it for now? We can add it back if necessary later.

**Comment by xukai92** [comment]
@yebai I think functions exported on https://github.com/yebai/Turing.jl/blob/master/src/Turing.jl#L76 are just functions from Mamba.jl - users who want to use them should import them from the Mamba.jl package - as we already make our `Chain` type a subtype of their `AbstractChain`, it will just work. How do you think?

**Comment by yebai** [comment]
Fixed by #301

**Comment by dcarrera** [comment]
The last comment says that this issue is fixed. Is this not incorporated in the latest release? I just installed Turing for the first time and I immediately this bug. I was trying to plot with PyPlot and I hit the conflict with Turing.
